core/templates/pages/about-page/about-page.controller.ts

```diff
@@ -79,15 +70,15 @@ angular.module('oppia').directive('aboutPage', [
 
             var hash = WindowRef.nativeWindow.location.hash.slice(1);
             if (hash === ctrl.TAB_ID_FOUNDATION || hash === 'license') {
-              activateTab(ctrl.TAB_ID_FOUNDATION);
+              ctrl.activeTabName = ctrl.TAB_ID_FOUNDATION;
             }
 
             if (hash === ctrl.TAB_ID_CREDITS) {
-              activateTab(ctrl.TAB_ID_CREDITS);
+              ctrl.activeTabName = ctrl.TAB_ID_CREDITS;
             }
 
             if (hash === ctrl.TAB_ID_ABOUT) {
-              activateTab(ctrl.TAB_ID_ABOUT);
+              ctrl.activeTabName = ctrl.TAB_ID_ABOUT;
             }
```

Comment:
Would just `ctrl.activeTabName = hash;` suffice here (with an exception for `'license'`)? Or if you want to check that the hash is in the `[ctrl.TAB_ID_TEACH, ctrl.TAB_ID_PLAYBOOK]` you can probably make a helper function and use it also on lines 79 to 83.

Reply:
Done.

Reply:
Can we get rid of the separate ifs then and do just `ctrl.activeTabName = hash`?

Reply:
@marianazangrossi What about this?

Reply:
I think I've already done it @vojtechjelinek This piece of code is outdated